.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
you might want to have a full .gitignore https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
added full gitignore as you suggested